src/kokkos/test/kokkos/TrackingRecHit2D_t.cc

```diff
@@ -0,0 +1,70 @@
+#include "KokkosCore/kokkosConfigCommon.h"
+#include "KokkosCore/kokkosConfig.h"
+
+#include "CondFormats/pixelCPEforGPU.h"
+#include "KokkosDataFormats/TrackingRecHit2DKokkos.h"
+
+namespace testTrackingRecHit2DKokkos {
+
+  template<typename MemorySpace>
+  void fill(Kokkos::View<TrackingRecHit2DSOAView, MemorySpace> hits) {
+
+    assert(hits.data());
+    auto &hits_ = *hits.data();
+
+    Kokkos::parallel_for("fill", Kokkos::RangePolicy<KokkosExecSpace>(0, 1024), KOKKOS_LAMBDA(const size_t i) {
```

Comment:
This is ok here, but for functions called in the "production" code the policies should take also the execution space object (that would be one of the function's parameters).

Reply:
Actually this might require the execution space object being passed in along
```suggestion
    Kokkos::parallel_for("fill", Kokkos::RangePolicy<KokkosExecSpace>(KokkosExecSpace(), 0, 1024), KOKKOS_LAMBDA(const size_t i) {
```
in order to be able to run the serial backend binaries (because then the possible internal `fence()` calls will be limited to the `KokkosExecSpace` instead of full `Kokkos::fence()` that would try to synchronize also the CUDA backend, that is not initialized for the serial backend binaries).